modules/local-caching-classloader/src/main/java/org/apache/accumulo/classloader/lcc/definition/Resource.java

```diff
@@ -18,37 +18,32 @@
  */
 package org.apache.accumulo.classloader.lcc.definition;
 
-import java.net.MalformedURLException;
 import java.net.URL;
 import java.util.Objects;
 
-public class Resource implements Comparable<Resource> {
+public class Resource {
 
-  private String location;
+  private URL location;
```

Comment:
This is good, I wasn't sure if gson supported serializing/deserializing URLs without an adapter.